### DISCUSSION TITLE
How to fine tune parameters of generated speech?

### DISCUSSION THREAD

**Discussion by penyuan** [coqui-ai/TTS#512] [General Q&A]
It's very exciting to discover coqui-ai/TTS and that it can synthesise speech from the included models out-of-the-box. For example, I've successfully used the built-in Tacotron2-DCA model to read some English text I have (using the instructions [here](https://github.com/coqui-ai/TTS#example-synthesizing-speech-on-terminal-using-the-released-models)). Thank you for maintaining this great project! I've got some beginner questions as an end-user (I'm not training any models -> I just want to use coqui to read text): 

1. How do I get a different voice? For example since my use case is in English, are there voices with British/American/others to choose from?
2. Are there ways to adjust the speed and pitch of the voice?
3. Is Speech Synthesis Markup Language (SSML) (or something equivalent) supported?

Generally speaking, my questions are around how to fine-tune the output. For example, the [IBM Watson TTS demo page](https://www.ibm.com/demos/live/tts-demo/self-service/home) has a bunch of adjustable parameters. I'd like to know how I can achieve something similar with coqui-ai/TTS.

Thanks!

**Comment by erogol** [comment]
Thanks for your interest in :frog: TTS :)

1. How do I get a different voice? For example, since my use case is in English, are there voices with British/American/others to choose from?
> Check the released models. These are all we can provide for now. 

2. Are there ways to adjust the speed and pitch of the voice?
> Not yet, but there is a discussion about it right now and some of our contributors try to make it work. Join in the conversation if you are interested. 

3 . Is Speech Synthesis Markup Language (SSML) (or something equivalent) supported?
> Not yet.

IBM is a big company, Coqui is a 2 months old startup. Hopefully, we get there with your help :).

**Reply by penyuan** [reply]
Thank you @erogol for responding so quickly! I didn't realize Coqui is only two-months old. Amazing work and thanks for keeping everything open source!

> Check the released models. These are all we can provide for now.

OK, thanks! But using the `--list_models` argument to get a list of models doesn't really tell me anything about what the voices are. Are there descriptions somewhere I can read? Also, should I be looking at the "Text-to-Spectrogram" or "Vocoder" models to find out what voices are available? And is it a matter of trying all their combinations (i.e. trial and error) or is there documentation somewhere other than the linked academic papers?

> IBM is a big company, Coqui is a 2 months old startup. Hopefully, we get there with your help :).

While I know Python my expertise is not deep learning or speech synthesis. But if there are other ways to help I'd be happy to help!
